scripts/prereq-check-enterprise.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+check_prereq()
+{
+    local dependency=$1
+    type "$dependency" >/dev/null 2>&1 || { echo >&2 "Mattermost Enterprise requires '$dependency' but it doesn't appear to be installed.  Aborting."; exit 1; }
+}
+
+echo "Checking enterprise prerequisites"
+
+check_prereq 'xmlsec1'
```

Comment:
I couldn't find the minimum version required. Our build environment docker files installs the latest available version.
https://github.com/mattermost/mattermost-server/blob/master/build/Dockerfile.buildenv